zeppelin-interpreter/src/main/java/org/apache/zeppelin/cluster/ClusterManagerServer.java

```diff
@@ -210,6 +208,14 @@ public BroadcastService getBroadcastService() {
         messagingService.registerHandler(CLUSTER_AUTH_EVENT_TOPIC,
             subscribeClusterAuthEvent, MoreExecutors.directExecutor());
 
+        HashMap<String, Object> meta = new HashMap<String, Object>();
+        String nodeName = getClusterNodeName();
+        meta.put(ClusterMeta.NODE_NAME, nodeName);
+        meta.put(ClusterMeta.SERVER_HOST, zeplServerHost);
+        meta.put(ClusterMeta.SERVER_PORT, raftServerPort);
+        meta.put(ClusterMeta.SERVER_START_TIME, new Date());
```

Comment:
We should not use old date api java.util.Date, either we use the new Date api in java 8 or joda

Reply:
This date field is only used to display the start time of the service and will not participate in the calculation.

Reply:
The old date api has issue for time zone.

Reply:
done.